holmes/core/supabase_dal.py

```diff
@@ -403,6 +404,69 @@ def get_issue_data(self, issue_id: Optional[str]) -> Optional[Dict]:
 
         return issue_data
 
+    def get_runbook_catalog(self) -> Optional[List[RobustaRunbookInstruction]]:
+        if not self.enabled:
+            return None
+
+        try:
+            res = (
+                self.client.table(RUNBOOKS_TABLE)
+                .select("*")
+                .eq("account_id", self.account_id)
+                .eq("subject_type", "RunbookCatalog")
+                .execute()
+            )
+            if not res.data:
+                return None
+
+            instructions = []
+            for row in res.data:
+                id = row.get("runbook_id")
+                symptom = row.get("symptoms")
+                title = row.get("subject_name")
+                if not id or not symptom:
+                    continue
```

Comment:
we don't have validation in the UI for symproms? maybe worth to add log here in case it happens?